### ISSUE TITLE
associate advisory times to NHC times

### ISSUE THREAD

**Issue by carolakaiser** [StormSurgeLive/StormReplay.com#7]
For some of the users of this tool like CERA, it might be helpful to associate the advisory times to the official NHC times 03, 09,15, 21Z.

**Comment by wwlwpd** [comment]
I agree. I think it makes sense to restrict the times to conform to NHC's norms.
